### PR TITLE
Added courses of Git, Django, Python, .NET, HTML/CSS, C, C++

### PR DESCRIPTION
**Pull request by TejasMate**
Renamed "Data Structure" section into "Algorithms & Data Structures". Created new section for Django, Git, .NET, HTML/CSS courses and added new courses in it. Added courses in existed section.

## Hacktoberfest notes:

- due to volume of submissions, we may not be able to review PRs that do not pass tests and do not have informative titles.
- please read our [contributing guidelines](/CONTRIBUTING.md)
- be sure to check the output of Travis-CI for linter errors
- if this is your first open source contribution, make sure it's not your last!

## What does this PR do?
Add Resource(s) | Remove Resource(s) | Add info | Improve Repo

### Checklist:
- [x] Not a duplicate
- [x] Included author(s) if appropriate
- [x] Lists are in alphabetical order
- [x] Needed indications added (PDF, access notes, under construction)

### CODE REVIEW

File: free-courses-hi.md [hunk 1]
```diff
@@ -1,46 +1,76 @@
...
 
 
 ### Python
 
-* [Python Tutorial For Beginners | Hindi (With Notes)](https://www.youtube.com/playlist?list=PLu0W_9lII9agICnT8t4iYVSZ3eykIAOME) - Harry
+* [Python Tutorial For Beginners | Hindi (With Notes)](https://www.youtube.com/playlist?list=PLu0W_9lII9agICnT8t4iYVSZ3eykIAOME) - Harry (Youtube)
+* [Intermediate/Advanced python Tutorials in Hindi](https://www.youtube.com/playlist?list=PLu0W_9lII9aiJWQ7VhY712fuimEpQZYp4) - Harry (Youtube)
+
+
+### Django
```
Comment (SethiPandi) [inline free-courses-hi.md:74]: This is a subsection under Python, could you give it h4 instead of h3?

```suggestion
#### Django
```

**Review by SethiPandi — CHANGES_REQUESTED** [review]
Please ensure all lists and categories remain in alphabetical order.

You can get more guidance on the formatting and conventions of this repository by checking out the [contribution guide](https://www.github.com/EbookFoundation/free-programming-books/tree/master/CONTRIBUTING.md).

File: free-courses-hi.md (hunk 1 above)
Comment (TejasMate) [reply free-courses-hi.md]: Removed outdated course & Made h4 heading to "Django".

**Comment by TejasMate** [comment]
Other free-courses-lang code sorted [.NET] with N in Index not with .(dot),  i did same.

Alphabetically .(dot) comes first before letter. So can i change ranks .NET as first or remains as it is (in N order).

**Comment by eshellman** [comment]
do not add '(YouTube)' see discussion at #4579. Also, when you change many listings it's hard to see what you've added.

**Pull request closed without merging**